introduction_to_applying_machine_learning/xgboost_customer_churn/xgboost_customer_churn.ipynb

```diff
@@ -243,7 +243,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We see several features that essentially have 100% correlation with one another.  Including these feature pairs in some machine learning algorithms can create catastrophic problems, while in others it will only introduce minor redundancy and bias.  Let's remove them anyway."
+    "We see several features that essentially have 100% correlation with one another.  Including these feature pairs in some machine learning algorithms can create catastrophic problems, while in others it will only introduce minor redundancy and bias.  Let's remove one of the feature in each highly correlated pairs: Day Charge from the pair with Day Mins, Night Charge from the pair with Night Mins, Intl Charge from the pair with Intl Mins:"
```

Comment:
Tiny change request:
"...one of the feature in each highly correlated pairs..." to "...one feature from each of the highly correlated pairs..."

Reply:
Looks good. Fixed in latest commit.